community/modules/scheduler/htcondor-configure/versions.tf

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 3.0, < 5.0"
```

Comment:
We have been back and forth about this before I believe :). I thought it was best to not specify a max version on non-root modules.

From [terraform docs](https://www.terraform.io/language/providers/requirements#best-practices-for-provider-versions): _ Do not use maximum-version constraints for modules you intend to reuse across many configurations, even if you know the module isn't compatible with certain newer versions_

If I am mistaken or you believe this shouldn't apply to us, let me know so I can stop bringing it up.

Reply:
Should be resolved.